CMakeLists.txt

```diff
@@ -6,5 +6,6 @@ set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 
+add_subdirectory(lib/spotify-json)
```

Comment:
There is the `EXCLUDE_FROM_ALL` flag for `add_subdirectory` to disable including subdirectory targets to the project's targets list.